kpi/002.adoc

```diff
@@ -71,5 +70,5 @@ metadata record (abstract, keywords, etc.).
 For a satellite product offering one main data parameter, the title will
 typically define which parameter is contained in the product, and from which
 instrument or instrument type it originates. For example, "AMSR-2 Sea Surface
-Temperature” or “SLSTR L1B radiances and brightness temperatures”.footnote:[WCMP Guidance (http://wis.wmo.int/file=3291)]
+Temperature” or “SLSTR L1B radiances and brightness temperatures”.footnote:[WCMP Guidance (https://wis.wmo.int/file=3291)]
```

Comment:
The link https://wis.wmo.int/file=3291  points to a draft document in wmosharepoint

Reply:
@amilan17 can you you point to us to the correct URL here?